conventions/services/recapitulatif.py

```diff
@@ -309,7 +311,9 @@ def collect_instructeur_emails(
     return instructeur_emails, submitted
 
 
-def send_email_instruction(convention_url, convention, instructeur_emails):
+def send_email_instruction(
+    convention_url, convention, bailleur, user, instructeur_emails
+):
     """
```

Comment:
Suggestion : on pourrait créer le `dict` `email_data` plus tôt et le réutiliser dans les deux fonctions plus bas 

```suggestion
  email_data={
    "convention_url": convention_url,
    "convention": str(convention),
    "bailleur": str(bailleur),
    "user": str(user),
  }
    """
```

Reply:
Et même question que plus bas : on ne pourrait pas récupérer le bailleur directement sur la convention plutôt que d'ajouter un paramètre à la fonction ?

Reply:
sauf qu'il sert à d'autres endroits avec d'autres données

Reply:
pas dans cette fonction si ? elle se termine ligne 358, dans les autres fonctions on ne sera plus dans le même contexte d'execution et la variable n'aura pas la même valeur

Reply:
ok